cypress/integration/inventory/call-number-browse.spec.js

```diff
@@ -0,0 +1,19 @@
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+
+describe('verify call number browse feature', () => {
+  beforeEach('navigate to inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C347902 Verify "Browse call numbers" option on the Instances tab', { tags: [TestTypes.smoke] }, () => {
+    InventorySearch.verifyKeywordsAsDefault();
+    InventorySearch.selectCallNumberBrowse();
+    InventorySearch.verifyCallNumberBrowseEmptyPane();
```

Comment:
"The label "Browse inventory" is displayed on the main pane" - can't see it. May be  bug or outdated test

Reply:
the same related with ""Browse for results entering a query or choosing a filter" on the main pane is displayed."

Reply:
These checks indide funtion verifyCallNumberBrowseEmptyPane:

```
verifyCallNumberBrowseEmptyPane() {
    const callNumberBrowsePane = Pane({ title: 'Browse inventory' });
    cy.expect(callNumberBrowsePane.exists());
    cy.expect(callNumberBrowsePane.has({ subtitle: 'Enter search criteria to start browsing' }));
    cy.expect(HTML(including('Browse for results entering a query or choosing a filter.')).exists());
  },
```